integration-tests/vertx-graphql/src/test/java/io/quarkus/vertx/graphql/it/VertxGraphqlTest.java

```diff
@@ -20,14 +20,22 @@
 
 @QuarkusTest
 class VertxGraphqlTest {
-
-    @Inject
-    Vertx vertx;
-
     public static int getPortFromConfig() {
         return ConfigProvider.getConfig().getOptionalValue("quarkus.http.test-port", Integer.class).orElse(8081);
     }
 
+    private static Vertx vertx;
+
+    @BeforeAll
+    public static void initializeVertx() {
+        vertx = Vertx.vertx();
+    }
+
+    @AfterAll
+    public static void closeVertx() {
+        vertx.close();
```

Comment:
You need:

```
CountDownLatch latch = new CountDownLatch(1);
vertx.close(() -> {
     latch.countDown();
});
latch.await();
```

Reply:
^ pseudo-code :-D

Reply:
Thanks, the other `vertx#close()` was to tempting for me. I'll change it :-)